### ISSUE TITLE
AMD Support

### ISSUE THREAD

**Issue by edsilv** [FNNDSC/ami#365]
### Description
Unable to load AMI using require.js (AMD).

#### Live Demo
https://github.com/edsilv/ami-amd-test

#### Steps to Reproduce
`npm start`

#### Expected Results
Logs AMI object.

#### Actual Results
Logs `undefined` here: https://github.com/edsilv/ami-amd-test/blob/master/index.html#L17

### Browsers Affected
- [x ] All
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari 9
- [ ] Safari 8
- [ ] IE 11

### Versions
- AMI: v0.27.0

**Comment by NicolasRannou** [comment]
@edsilv yes what you suggest would be ideal but we just didn't have time to look into :(
PRs welcome :)
